Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkTransformationManager.java

```diff
@@ -209,4 +209,138 @@ public ITransformedNetwork getSearchTransformedNetwork(String projectId, String
 
         return networkList;
     }
+    
+    
+    @Override
+    public ITransformedNetwork getSearchTransformedNetwork(List<String> projectIds, String conceptId)
```

Comment:
don't use the same method name if the method does something else. Choose a name that describes what the method it doing.